src/components/DeeplinkWrapper/index.website.js

```diff
@@ -96,7 +101,7 @@ class DeeplinkWrapper extends PureComponent {
                     return;
                 }
                 iframe.parentNode.removeChild(iframe);
-            }, 100);
+            }, 0);
```

Comment:
Why do actually need this `setTimeout()`? The explanation above is not very satisfying...

>we need to give this iframe some time for it to do what it needs to do

What does it mean?